tools/logger/CMakeLists.txt

```diff
@@ -1,5 +1,7 @@
 # SPDX-License-Identifier: BSD-3-Clause
 
```

Comment:
Can you also put the warning in the commit message. It looks out of context.

Reply:
Added.

> It looks out of context.

A little, but it's quite obvious problem and solution so I didn't create another PR for this, especially when we have some troubles with merging and number of PR grow. After merge there won't be any difference between two vs one PR) Moreover it's not any hotfix, so there is no need to introduce it very fast.